docs/lsp-clients.json

```diff
@@ -334,6 +334,14 @@
     "installation": "pip install fortls",
     "debugger": "Yes"
   },
+  {
+    "name": "futhark",
+    "full-name": "Futhark",
+    "server-name": "futhark-lsp",
+    "server-url": "https://github.com/diku-dk/futhark"
+    "installation": "A part of the compiler since 0.21.9",
+    "debugger": "N/A"
```

Comment:
Maybe change it to not available just for consistency.

Reply:
Both N/A and Not Available are in there

Reply:
Anyway, fixed.

But the file is already very inconsistent

Reply:
Yeah, you are right. I was using a mobile device, so I couldn't see the whole file.

Sorry, you have to deal with those legacy changes. I prefer some consistency rather than no consistency. 🤔